python/ray/data/tests/test_map.py

```diff
@@ -530,10 +530,10 @@ def __call__(self, x):
         fn_constructor_kwargs={"b": put(2)},
     )
     ds_list = ds2.take()
-    values = [s["one"] for s in ds_list]
+    values = sorted([s["one"] for s in ds_list])
     assert values == [2, 4, 6]
     values = [s["two"] for s in ds_list]
-    assert values == [4, 6, 8]
+    assert values == sorted([4, 6, 8])
```

Comment:
we don't need to do this, right?

Reply:
Oops, I missed this comment. Let me create a new PR.

Reply:
https://github.com/ray-project/ray/pull/38422